app/views/shared/_top_nav.html.erb

```diff
@@ -1,19 +1,17 @@
-<div class="custom-nav" id="nav-custom">
+<!-- commenting for now until we add the booking component to homepage and decide
```

Comment:
I would recommend deleting this completely. We saw the problem with commented out embedded ruby during the live screencast.